### PR TITLE
Feat parallel repo list

### PR DESCRIPTION
**Pull request by fredbi**

### CODE REVIEW

File: pkg/core/repo_list_test.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+			},
+		},
+		{
+			name:     happyWithBatches,
+			expected: expectedRepoBatchFixture,
+		},
+	}
+}
+
+func buildRepoYaml(repo string) string {
```
Comment (kerneltime) [inline pkg/core/repo_list_test.go]: Why no use the model pkg? I am worried about silent failures as this format is not based on the actual deserialization.

File: cmd/datamon/cmd/bundle_list.go [hunk 2]
```diff
@@ -37,8 +36,8 @@ var BundleListCommand = &cobra.Command{
 			return
 		}
 		err = core.ListBundlesApply(params.repo.RepoName, remoteStores.meta, applyBundleTemplate,
-			core.ConcurrentBundleList(params.core.ConcurrencyFactor),
-			core.BundleBatchSize(params.core.BatchSize))
+			core.ConcurrentList(params.core.ConcurrencyFactor),
```
Comment (kerneltime) [inline cmd/datamon/cmd/bundle_list.go]: in ```cmd/repo_list.go``` add ```addConcurrencyFactorFlag(repoList)```

File: pkg/core/repo_list_test.go (hunk 1 above)
Comment (fredbi) [reply pkg/core/repo_list_test.go]: Good idea. Have some issues with timestamps as yaml etc, though. Testing that right now

File: cmd/datamon/cmd/bundle_list.go (hunk 2 above)
Comment (fredbi) [reply cmd/datamon/cmd/bundle_list.go]: yep forgot that. True

**Pull request merged**